README.md

```diff
@@ -10,6 +10,14 @@ If your platform isn't yet supported, you can create the native binary from
 the latest upstream clang sources, make sure it is stripped and optimized
 (should be about 1.4MB as of mid-2015) and send a pull request to add it.
 
+## Globbing files
+
+    $ clang-format --glob=folder/**/*.js
+
+This will run `clang-format` once per file result, and show the total
+formatted files at the end.
+See [node-glob](https://github.com/isaacs/node-glob) for globbing semantics.
```

Comment:
Just realized that we don't afford a way to pass options - I was just trying to write up an example and need `exclude`...